src/main/java/seedu/address/commons/events/ui/PersonHasBeenDeletedEvent.java

```diff
@@ -0,0 +1,23 @@
+package seedu.address.commons.events.ui;
+
+import seedu.address.commons.events.BaseEvent;
+import seedu.address.model.person.ReadOnlyPerson;
+
+//@@author nbriannl
+
+/**
+ * Represents a deletion of a person.
+ */
+public class PersonHasBeenDeletedEvent extends BaseEvent {
```

Comment:
`PersonHasBeenDeletedEvent` `PersonHasBeenModifiedEvent` are events Person Info Panel will look out for.